docs/clients/power_safety.rst

```diff
@@ -112,36 +111,36 @@ Message Table
-+--------+-----------------------+----------------+-----------+------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| 13     | temperature_coil_low  | get, set, save | float     | degC | Minimum threshold value for coil temperature                                                                                                                                                                                                       |
-+--------+-----------------------+----------------+-----------+------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| 14     | temperature_coil_high | get, set, save | float     | degC | Maximum threshold value for coil temperature                                                                                                                                                                                                       |
-+--------+-----------------------+----------------+-----------+------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++--------+-----------------------+----------------+-----------+-----------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Sub ID | Short Name            | Access         | Data Type | Unit            | Note                                                                                                                                                                                                                                               |
++========+=======================+================+===========+=================+====================================================================================================================================================================================================================================================+
+| 0      | fault_now             | get            | uint8     | Number [0, 255] | The fault(s) that are currently triggering                                                                                                                                                                                                         |
```

Comment:
Maybe the unit for fault_now, fault_ever, and fault_latching would be better as "Bitmask". They are numbers, but more specifically they are acting as bitmasks